src/runtime/lpython_builtin.py

```diff
@@ -764,6 +764,72 @@ def _lpython_str_upper(x: str) -> str:
     return res
 
 
+@overload
+def _lpython_str_isalpha(s: str) -> bool:
+    char: str
```

Comment:
Here is the cause for the CI failure: https://github.com/lcompilers/lpython/issues/2359

I think renaming it or fixing the bug would resolve the CI failure.